### PR TITLE
Remove unused imports

### PR DESCRIPTION
**Pull request by ceh**

### CODE REVIEW

File: mypy/checkexpr.py [hunk 1]
```diff
@@ -18,9 +18,8 @@
 from mypy.types import (
     Type, AnyType, CallableType, Overloaded, NoneTyp, TypeVarDef,
     TupleType, TypedDictType, Instance, TypeVarType, ErasedType, UnionType,
-    PartialType, DeletedType, UnboundType, UninhabitedType, TypeType, TypeOfAny,
-    true_only, false_only, is_named_instance, function_type, callable_type, FunctionLike,
-    get_typ_args, StarType
+    PartialType, DeletedType, UninhabitedType, TypeType, TypeOfAny, true_only,
+    false_only, is_named_instance, function_type, callable_type, FunctionLike, StarType
```
Comment (gvanrossum) [inline mypy/checkexpr.py]: Can you add a trailing comma? In lists like this that's how we like to format them.

File: mypy/stats.py [hunk 2]
```diff
@@ -1,10 +1,9 @@
 """Utilities for calculating and reporting statistics about types."""
 
 import os.path
-import typing
 
 from collections import Counter
-from typing import Dict, List, cast, Optional
+from typing import cast, Counter, Dict, List, Optional
```
Comment (gvanrossum) [inline mypy/stats.py]: This doesn't seem right. It overrides the import of `Counter` on the previous line. I think there's a reason why the code isn't doing that.

Comment (gvanrossum) [inline mypy/stats.py]: This doesn't seem right. It overrides the import of `Counter` on the previous line. I think there's a reason why the code did what it did.

File: mypy/checkexpr.py (hunk 1 above)
Comment (ceh) [reply mypy/checkexpr.py]: Fixed.

File: mypy/stats.py (hunk 2 above)
Comment (ceh) [reply mypy/stats.py]: Ah, you are correct. I missed this. Thanks for the helpful comment.

**Comment by gvanrossum** [comment]
Thanks again!

**Pull request merged**